network/src/network.rs

```diff
@@ -1261,6 +1262,28 @@ impl NetworkController {
             .ban_session(&self.p2p_control, peer_index, duration, reason);
     }
 
+    /// disconnect peers with matched peer_ip or peer_ip_network, eg: 192.168.0.2 or 192.168.0.0/24
+    fn disconnect_peers_in_ip_range(&self, address: IpNetwork, duration: Duration, reason: &str) {
+        self.network_state.with_peer_registry(|reg| {
+            reg.peers().iter().for_each(|(peer_index, peer)| {
+                if let Some(addr) = multiaddr_to_socketaddr(&peer.connected_addr) {
+                    if address.contains(addr.ip()) {
+                        let _ = disconnect_with_message(
+                            &self.p2p_control,
+                            *peer_index,
+                            &format!(
+                                "Ban peer {:?} for {} seconds, reason: {}",
```

Comment:
this message is incorrectly, the params `ban_until` is a timestamp not duration.

Reply:
Keep it simple, i rewrite message as "Ban peer {}, reason: {}"